app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -60,3 +60,10 @@
   <li>Use the link to their website to find out further details and how to apply.</li>
   <li>Maximise the chances of your application succeeding with <a href="/explore-teaching-advisers">free one-to-one support from an explore teaching adviser</a>.</li>
 </ol>
+
+<h2> Apply for a teaching internship</h2>
```

Comment:
do we need this extra h2 @philspray ? (can remove retrospectively though!!)

Reply:
Hmm just looked at the archive - maybe it needs an h3 intro eg previously we had an h3 of:

`Find a teaching internship in your region`